docs/migrating-from-onenote.md

```diff
@@ -0,0 +1,20 @@
+# Migrating from OneNote
+
+This guide mostly duplicates the instructions at the repo for the PowerShell [script](https://github.com/nixsee/ConvertOneNote2MarkDown).
+
```

Comment:
Perhaps we should add some details of what the script does/allows for? There's a lot to choose from in the Summary section of the script Readme.